msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS-FrameworkList.xml.in

```diff
@@ -2,4 +2,48 @@
 <FileList Name="Xamarin.iOS" TargetFrameworkDirectory="%TargetFrameworkDirectory%">
   <File AssemblyName="System.Buffers" Version="4.0.99.0" />
   <File AssemblyName="System.Memory" Version="4.0.99.0" />
```

Comment:
why were those added ?
and why don't they include tokens and processor arch ? i.e. are those needed ?

Reply:
> why were those added ?

No clue, see https://github.com/xamarin/xamarin-macios/commit/6f2ebedb74fe8da250c8f53b61ffae2eb86fc711#diff-e801bb766cbaad95b50b1487b865f971?

> and why don't they include tokens and processor arch ?

The people adding that code didn't add that info and those assemblies are not in our "normal" assembly folder. To be fair I haven't looked for them too much, maybe they're in some other folder and I could update the test to report them properly...

> are those needed ?

Mikayla said so (:

Reply:
@spouliot `/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/Xamarin.iOS/Facades`. I don't know if Xamarin.iOS/Mac should list everything in that Facades folder in `Xamarin.iOS-FrameworkList.xml.in`.